package.json

```diff
@@ -1,81 +1,84 @@
 {
-    "name": "featureboard-sdks",
```

Comment:
Just need to save this file, formatting has been messed up (nx doesn't respect indent settings when formatting, it's annoying)

Reply:
Resolved